src/integrations/prefect-kubernetes/prefect_kubernetes/observer.py

```diff
@@ -110,15 +127,24 @@ async def _replicate_pod_event(  # pyright: ignore[reportUnusedFunction]
     if event_type is None:
         if orchestration_client is None:
             raise RuntimeError("Orchestration client not initialized")
+
+        # Use the Kubernetes event timestamp for the filter to avoid "Query time range is too large" error
+        event_filter = EventFilter(
+            event=EventNameFilter(name=[f"prefect.kubernetes.pod.{phase.lower()}"]),
+            resource=EventResourceFilter(
+                id=[f"prefect.kubernetes.pod.{uid}"],
+            ),
+            **(
+                {"occurred": EventOccurredFilter(since=k8s_event_time)}
+                if k8s_event_time
+                else {}
+            ),
```

Comment:
I still think this needs to have _some_ `occurred` filter, how about `now - 1h`?

Reply:
(as the fallback)